packages/reports/addon/components/asset-owner.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright 2022, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import Component from '@glimmer/component';
+import moment from 'moment';
+
+interface ReportUserArgs {
+  user: string;
+  updatedOn: string;
+}
+
+export default class ReportUser extends Component<ReportUserArgs> {
+  get updatedon(): string {
+    return 'Last Modified on ' + moment(this.args.updatedOn).format('MMM');
```

Comment:
We should have more than just the month, this format is probably better `'Jan 28, 2022 12:01 PM'`
```suggestion
    return 'Last Modified on ' + moment(this.args.updatedOn).format('lll');
```